<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>667</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CuteInterceptor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../.designer/backup/icon.png</normaloff>../../.designer/backup/icon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonStart">
    <property name="geometry">
     <rect>
      <x>1065</x>
      <y>10</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QComboBox" name="fileBox">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>10</y>
      <width>946</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="maxCount">
     <number>100</number>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
    </property>
    <property name="minimumContentsLength">
     <number>300</number>
    </property>
   </widget>
   <widget class="PlotWidget" name="signal_plot">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>85</y>
      <width>701</width>
      <height>206</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="foregroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="interactive">
     <bool>true</bool>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>0.000000000000000</width>
      <height>0.000000000000000</height>
     </rectf>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="transformationAnchor">
     <enum>QGraphicsView::AnchorViewCenter</enum>
    </property>
   </widget>
   <widget class="QLabel" name="deviceName">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>10</y>
      <width>66</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>File</string>
    </property>
   </widget>
   <widget class="QLabel" name="deviceName_2">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>340</y>
      <width>106</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0 - Horizontal</string>
    </property>
   </widget>
   <widget class="QLabel" name="deviceName_3">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>370</y>
      <width>106</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>2 - Vertical</string>
    </property>
   </widget>
   <widget class="QLabel" name="deviceName_4">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>340</y>
      <width>106</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1 - Diagonal</string>
    </property>
   </widget>
   <widget class="QLabel" name="deviceName_5">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>370</y>
      <width>136</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>3 - Anti-Diagonal</string>
    </property>
   </widget>
   <widget class="QLabel" name="angle_3">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>55</y>
      <width>311</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Signal</string>
    </property>
   </widget>
   <widget class="QLabel" name="angle_4">
    <property name="geometry">
     <rect>
      <x>725</x>
      <y>55</y>
      <width>311</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Signal: Height Clusters</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="histo_plot">
    <property name="geometry">
     <rect>
      <x>725</x>
      <y>85</y>
      <width>536</width>
      <height>206</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="foregroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>0</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
    <property name="interactive">
     <bool>true</bool>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>0.000000000000000</width>
      <height>0.000000000000000</height>
     </rectf>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="transformationAnchor">
     <enum>QGraphicsView::AnchorViewCenter</enum>
    </property>
   </widget>
   <widget class="QLabel" name="angle_5">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>300</y>
      <width>386</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Assign Cluster Names to Polarisation</string>
    </property>
   </widget>
   <widget class="QLabel" name="angle_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>311</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Decoded Signal</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_0">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>340</y>
      <width>66</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>370</y>
      <width>66</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>E</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>340</y>
      <width>66</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>370</y>
      <width>66</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDecode">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>370</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Decode!</string>
    </property>
   </widget>
   <widget class="QLabel" name="deviceName_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>445</y>
      <width>126</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Polarisation:</string>
    </property>
   </widget>
   <widget class="QLabel" name="raw_key">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>445</y>
      <width>1066</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="text_ClusterReport">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>285</y>
      <width>646</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="deviceName_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>515</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Mask(Hex):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineMask">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>520</y>
      <width>491</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="masked_signal">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>545</y>
      <width>1071</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="deviceName_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Masked Key(Bin):</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonApplyMask">
    <property name="geometry">
     <rect>
      <x>685</x>
      <y>520</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QLabel" name="raw_key_hex">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>490</y>
      <width>1061</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="deviceName_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>126</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Raw Key (Hex):</string>
    </property>
   </widget>
   <widget class="QLabel" name="raw_key_bin">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>467</y>
      <width>1066</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="deviceName_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>467</y>
      <width>126</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Raw Key (Bin):</string>
    </property>
   </widget>
   <widget class="QLabel" name="deviceName_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Masked Key(Hex):</string>
    </property>
   </widget>
   <widget class="QLabel" name="masked_signal_hex">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>575</y>
      <width>1076</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="deviceName_6">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>340</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Noise</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>340</y>
      <width>66</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
